pyanaconda/modules/boss/installation.py

```diff
@@ -128,24 +127,20 @@ def _copy_post_script_logs(self):
 
     def _dump_journal(self):
         """Dump journal from the installation environment"""
-        with open(join_paths(self._sysroot, ANACONDA_LOG_DIR, "journal.log"), "w") as logfile:
+        temp = "/tmp/anaconda-journal-dump"
+        with open(temp, "w") as logfile:
             execWithRedirect("journalctl", ["-b"], stdout=logfile)
-
-    def _chmod_logs(self):
-        """Set access bits"""
-        items = glob.glob(join_paths(self._sysroot, ANACONDA_LOG_DIR, '*'))
-        for item in items:
-            os.chmod(item, 0x0600)
+        self._copy_file_to_sysroot(temp, join_paths(ANACONDA_LOG_DIR, "journal.log"))
 
     def _copy_kickstart(self):
         """Copy input kickstart file"""
         if conf.target.can_copy_input_kickstart:
             log.info("Copying input kickstart file.")
             self._copy_file_to_sysroot(
                 "/run/install/ks.cfg",
-                "/root/original-ks.cfg"
+                "/root/original-ks.cfg",
+                0o0700
```

Comment:
Is the change of the permission expected? I am not sure where it comes from.

Reply:
IIRC the bits became different with the python code doing the copying, so I added this to compensate.

Reply:
Honestly this seems like a bug to me. Why we want to have executable bit for kickstart. I can't see a reason for that.
So, or so, it doesn't belong to this changeset.

Reply:
Blah, you are right. I wonder where did that 0700 come from then. Let's remove it, and with it the parameter, etc.